data/json/recipes/other/tool.json

```diff
@@ -134,9 +134,8 @@
     "type": "recipe",
     "category": "CC_OTHER",
     "subcategory": "CSC_OTHER_TOOLS",
-    "skill_used": "fabrication",
-    "difficulty": 4,
-    "skills_required": [ "fabrication", 4 ],
+    "skill_used": "electronics",
+    "difficulty": 3,
```

Comment:
Decided to look at the recipe of the coffeemaker as it is more or less a kitchen appliance, and modified the recipe to be in line with other already-in-game kitchen appliance. 
Althou I consider the foodprocessor to be more simpler than the coffeemaker, as it is just a blender but with different settings.